### DISCUSSION TITLE
How to add new colorschemes to Konsole profile?

### DISCUSSION THREAD

**Discussion by NovaViper** [nix-community/plasma-manager#131] [Q&A] [answered]
I saw in the module options for Konsole profiles that there's a colorScheme option https://github.com/pjones/plasma-manager/blob/5af3d8f504d1d63aef73eda65c7a946c3fe3ac7a/modules/apps/konsole.nix#L15-L21

But how do you actually install the themes onto your system? There's no specific NixOS package for Konsole themes. In my case, I'm trying to install the [Dracula theme for Konsole](https://github.com/dracula/konsole/).

**Comment by magnouvean** [comment, marked as answer]
You can place colorscheme files in `~/.local/share/konsole`. In fact I myself have exactly dracula colorscheme in [my config](https://codeberg.org/magnouvean/nixos-configs/src/commit/edaf196db2094ce5a445e47414c994fa5e8fec16/home/plasma.nix), so feel free to look for inspiration there.

**Reply by NovaViper** [reply]
Thank you!
